src/App.js

```diff
@@ -1,24 +1,89 @@
-import React, { Component } from 'react';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import "./grid.css";
+import weather from "./data/weather.json";
+import Search from "./Components/Search";
+import CurrentWeather from "./Components/CurrentWeather";
+import Upcoming from "./Components/Upcoming";
 
 class App extends Component {
-
   constructor(props) {
     super(props);
     this.state = {
+      weather: weather,
+      city: "London",
+      err: ""
     };
   }
+  //json data^
+  componentDidMount() {
+    this.handleClick();
+  }
+
+  changeHandler = event => {
+    this.setState({
+      city: event.target.value
+    });
+  };
+
+  handleClick = () => {
+    console.log("i've been clicked");
+    fetch(
+      `http://api.openweathermap.org/data/2.5/forecast?q=${
+        this.state.city
+      }&cnt=8&units=metric&appid=${"a75d5e4c8cb937be92dab81195a1c637"}`
+    )
+      .then(res => res.json())
+      .then(res => {
+        console.log("anything", res);
+        if (res.cod === "200") {
+          this.setState({
+            weather: res
+          });
+        }
+        if (res.cod !== "200") {
+          this.setState({
+            err: "invalid city name"
+          });
+        }
+      })
+      .catch(err => {
+        if (err) {
+          this.setState({
+            err: "invalid city name"
+          });
+        }
+      });
+  };
+
+  //response: what the user typed.
```

Comment:
I'd remove this comment because it's wrong. If you mean the response from the API, then that is not what the user typed, but a large JSON object with weather data in it!